portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,27 +26,35 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  List<String> comments = new ArrayList<>();
```

Comment:
This can be private final List<>...
private because, it only need to accessed within this class
final because comments variable will not change (the content of the array will, but the you will not reassign it after this).